From 69f55ad1d9f7becbf94762d69884fea1258e1113 Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Tue, 5 May 2015 14:33:35 +0800
Subject: [PATCH] MLK-10818-2 ARM: dts: add audio route configuration for
 imx6ul-evk board

add audio route configuration for imx6ul-evk board.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
---
 arch/arm/boot/dts/imx6ul-evk.dts | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ul-evk.dts b/arch/arm/boot/dts/imx6ul-evk.dts
index d158bfd..46926de 100755
--- a/arch/arm/boot/dts/imx6ul-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-evk.dts
@@ -53,6 +53,23 @@
 		 */
 		hp-det = <3 0>;
 		hp-det-gpios = <&gpio5 4 0>;
+		audio-routing =
+			"Headset Jack", "HP_L",
+			"Headset Jack", "HP_R",
+			"Ext Spk", "SPK_LP",
+			"Ext Spk", "SPK_LN",
+			"Ext Spk", "SPK_RP",
+			"Ext Spk", "SPK_RN",
+			"LINPUT2", "Hp MIC",
+			"LINPUT3", "Hp MIC",
+			"RINPUT1", "Main MIC",
+			"RINPUT2", "Main MIC",
+			"Hp MIC", "MICB",
+			"Main MIC", "MICB",
+			"CPU-Playback", "ASRC-Playback",
+			"Playback", "CPU-Playback",
+			"ASRC-Capture", "CPU-Capture",
+			"CPU-Capture", "Capture";
 	};
 
 	spi4 {
-- 
1.8.0

